src/OpenTelemetry.Exporter.Zipkin/README.md

```diff
@@ -26,19 +26,29 @@ take precedence over the environment variables.
 
 ### Configuration using Properties
 
+* `BatchExportProcessorOptions`: Configuration options for the batch exporter.
+  Only used if ExportProcessorType is set to Batch.
+
+* `Endpoint`: URI address to receive telemetry (default
+  `http://localhost:9411/api/v2/spans`).
+
+* `ExportProcessorType`: Whether the exporter should use [Batch or Simple
+  exporting
+  processor](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/trace/sdk.md#built-in-span-processors).
+
+* `HttpClientFactory`: A factory function called to create the `HttpClient`
+  instance that will be used at runtime to transmit spans over HTTP. See
+  [Configure HttpClient](#configure-httpclient) for more details.
+
+* `MaxPayloadSizeInBytes`: Maximum payload size - for .NET versions **other**
+   than 4.5.2 (default 4096).
```

Comment:
Isn't `4.6.1` the lowest version we support now. I guess this comment is from when it was supported and we no longer need to call it out

Reply:
https://github.com/open-telemetry/opentelemetry-dotnet/pull/2748